worldedit-bukkit/src/main/java/com/sk89q/worldedit/bukkit/BukkitBlockCommandSender.java

```diff
@@ -55,6 +57,7 @@ public BukkitBlockCommandSender(WorldEditPlugin plugin, BlockCommandSender sende
         checkNotNull(plugin);
         checkNotNull(sender);
 
+        this.audience = BukkitAudiences.create(plugin).audience(sender);
```

Comment:
You should have one instance of `BukkitAudiences.create(plugin)` cached and re-use the created factory to create your audiences.